lms/templates/courses_list.html

```diff
@@ -7,6 +7,20 @@
 
     % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
       <section class="courses">
+        <ul class="courses-listing journal-list">
```

Comment:
**Plugin required:** The Course listing page needs to be pluggable.
**Question:** Is this considered to be part of the student dashboard or a separate page altogether?